source/codegen/metadata/nirfmxwlan/functions.py

```diff
@@ -70,6 +70,142 @@
         ],
         'returns': 'int32'
     },
+    'AnalyzeNWaveformsIQ': {
+        'parameters': [
+            {
+                'direction': 'in',
+                'grpc_name': 'instrument',
+                'name': 'instrumentHandle',
+                'type': 'niRFmxInstrHandle'
+            },
+            {
+                'direction': 'in',
+                'name': 'selectorString',
+                'type': 'char[]'
+            },
+            {
+                'direction': 'in',
+                'name': 'resultName',
+                'type': 'char[]'
+            },
+            {
+                'direction': 'in',
+                'name': 'x0',
+                'size': {
+                    'mechanism': 'len',
+                    'value': 'arraySize'
+                },
+                'type': 'float64[]'
+            },
+            {
+                'direction': 'in',
+                'name': 'dx',
+                'size': {
+                    'mechanism': 'len',
+                    'value': 'arraySize'
+                },
+                'type': 'float64[]'
+            },
+            {
+                'direction': 'in',
+                'name': 'iq',
+                'size': {
+                    'mechanism': 'two-dimension',
+                    'value': 'iqSize'
+                },
+                'type': 'NIComplexSingle[]'
+            },
+            {
+                'direction': 'in',
+                'grpc_name': 'iq_lengths',
```

Comment:
I think these should be `iq_sizes`/etc not lengths. lengths is a fine name, but RFmx uses `size` throughout the API. And sizes is also closer to the original name.

Reply:
These have been updated in Scrapigen and the exports moved to apply here